### PR TITLE
A0-2842: Add copy icon

### PR DESCRIPTION
**Pull request by youPickItUp**

### CODE REVIEW

File: packages/extension-ui/src/components/Address.tsx [hunk 1]
```diff
@@ -504,9 +506,13 @@ export default styled(Address)(
   }
 
   .fullAddress {
+    display: flex;
+    align-items: center;
```
Comment (jalooc) [inline packages/extension-ui/src/components/Address.tsx:510]: Not better `space-between`? Wont' `center` cut away too much of the the address?

File: packages/extension-ui/src/components/Address.tsx [hunk 2]
```diff
@@ -504,9 +506,13 @@ export default styled(Address)(
...
+    column-gap: 4px;
+
     overflow: hidden;
     text-overflow: ellipsis;
-    max-width: 162px;
+    max-width: 178px;
```
Comment (jalooc) [inline packages/extension-ui/src/components/Address.tsx:515]: Sorry for being to lazy to run it on my own, but won't that break some views?

**Review by jalooc — COMMENTED** [review]
Looks good providing that we want the icon to be stationary instead of sliding. Just 2 small doubts to resolve.

File: packages/extension-ui/src/components/Address.tsx (hunk 2 above)
Comment (youPickItUp) [reply packages/extension-ui/src/components/Address.tsx:515]: I couldn't find any

File: packages/extension-ui/src/components/Address.tsx (hunk 1 above)
Comment (youPickItUp) [reply packages/extension-ui/src/components/Address.tsx:510]: I center them vertically -- it shouldn't impact available width

Comment (jalooc) [reply packages/extension-ui/src/components/Address.tsx:510]: Of course, my bad 🤦

**Review by jalooc — APPROVED** [review]

**Pull request merged**